datetime/src/main/java/com/afollestad/materialdialogs/datetime/DatePickerExt.kt

```diff
@@ -37,6 +38,9 @@ fun MaterialDialog.datePicker(
   minDate?.let {
     getDatePicker().minDate = minDate.timeInMillis
   }
+  maxDate?.let {
```

Comment:
These let's are weird. They are not designed to be a funny alias for `!= null`. It should either be:

```kotlin
maxDate?.let {
    getDatePicker().maxDate = it.timeInMillis
}
```

or

```kotlin
if(maxDate != null) {
    getDatePicker().maxDate = maxDate.timeInMillis
}
```

Reply:
I agree with you. We should use "it" in let block, but I added a copy of the functionality with minimal changes for author. I have not a point to fix all wrong let's in this project :)